.gitlab-ci.yml

```diff
@@ -169,6 +169,7 @@ test:rocm_install:
     # (with heavy libraries, tools etc. that also require GUI and other packages)
     - apt-get download hip-nvcc
     - $SUDO_CMD dpkg -i --ignore-depends=cuda hip*.deb
+    - $SUDO_CMD ln -s $ROCM_PATH /opt/rocm
```

Comment:
Why isn't the soft link working for you? It should be created.

Reply:
The softlink only not working when we use cuda image:
https://github.com/ROCmSoftwarePlatform/hipCUB/blob/dependency_handle_improve/.gitlab-ci.yml#L155-L174

When we install only hip-base like you can see in the CI script, the hipconfig is not in the path without the softlink.

Reply:
could you install rocm-dev? If that is not possible, a ticket should be raised.